ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorThread.java

```diff
@@ -61,6 +61,13 @@ public abstract class CompactorThread extends Thread implements Configurable {
   protected String hostName;
   protected String runtimeVersion;
 
+  //Time threshold for compactor thread log
+  //In milliseconds:
+  protected Integer MAX_WARN_LOG_TIME = 1200000; //20 min
```

Comment:
should this be configurable?

Reply:
Had a discussion with @veghlaci05 regarding the same and the conclusion was to keep this hardcoded for now, not to increase the number of configurable.
@veghlaci05 can you share more details here.

Reply:
I think it's not worth to introduce a new config value for this logging threshold, however I'm not strictly stick to it. We can make it configurable if you think it could be useful. 
@deniskuzZ, what do you think?

Reply:
ok, let's keep it hardcoded for now